libkodimote/audioplayer.cpp

```diff
@@ -37,3 +37,14 @@ Playlist * AudioPlayer::playlist() const
 {
     return m_playList;
 }
+
+void AudioPlayer::open(const PlaylistItem &item, bool resume)
+{
+    if (resume) {
+        qDebug() << "resume is not supported for AudioPlayer";
+    }
+
+    m_playList->clear();
+    m_playList->addItems(item);
+    playItem(0);
```

Comment:
hmm.. maybe this comment is on the wrong place. In any case, with this branch, if I go to Music -> Artists -> Albums -> SomeAlbum and then click on the 5th track in there, I expect the complete album to be added to the playlist and track number 5 to start playing. Works with master, doesn't work with this branch.